gslib/utils/copy_helper.py

```diff
@@ -288,6 +288,11 @@
              if CheckMultiprocessingAvailableAndInit().is_available else None))
 suggested_sliced_transfers_lock = parallelism_framework_util.CreateLock()
 
+COMMON_EXTENSION_RULES = {
+    '.md': 'text/markdown',
```

Comment:
Is there a bug related to markdown?

Reply:
There is, I'll send it to you somewhere else.